managed/ui/src/components/config/Security/KeyManagementConfiguration.js

```diff
@@ -331,6 +338,12 @@ class KeyManagementConfiguration extends Component {
                   disabled={false}
                   btnType="submit"
                 />
+                <YBButton
+                  btnText={'Cancel'}
+                  btnClass={'btn btn-orange'}
+                  disabled={false}
+                  onClick={this.showListView}
+                />
```

Comment:
- no need to wrap static text into `{}` as it's not an expression
- boolean attribute with false value could be simply omitted
```jsx
<YBButton
  btnText="Cancel"
  btnClass="btn btn-orange"
  onClick={this.showListView}
 />
```

Reply:
Done